cmd/meroxa/root/apps/common.go

```diff
@@ -2,15 +2,40 @@ package apps
 
 import (
 	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
 	"os/exec"
+	"path"
+	"time"
 
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/meroxa/cli/cmd/meroxa/global"
 	"github.com/meroxa/cli/log"
+	turbine "github.com/meroxa/turbine/deploy"
+)
+
+const (
+	dockerHubUserNameEnv    = "DOCKER_HUB_USERNAME"
+	dockerHubAccessTokenEnv = "DOCKER_HUB_ACCESS_TOKEN" // nolint:gosec
+	goLang                  = "golang"
+	javaScript              = "javascript"
+	nodeJS                  = "nodejs"
 )
 
 func buildGoApp(ctx context.Context, l log.Logger, appPath, appName string, platform bool) error {
 	var cmd *exec.Cmd
+
+	if appName != "" {
+		appName = appPath
+	}
```

Comment:
Looks like we don't use `appPath` anywhere. We should remove it (from function args too).